### DISCUSSION TITLE
ERO does not contain every property

### DISCUSSION THREAD

**Discussion by CarterBeauBenson** [CommonCoreOntology/CommonCoreOntologies#392] [Announcements]
For example, the Agent Ontology adds "is agent in." Why is it not the case that not all object properties are contained within ERO?

**Comment by alanruttenberg** [comment]
Be careful of going the route of RO, which became an unwieldy ontology without a well-defined scope. I'm not going to argue for any specific relationship not being in ERO, but have had discussions with other CCOers about not adopting RO's attitude and for the idea that relations that are domain specific reasonably belong in the domain ontology.
Seems perfectly reasonable that AgentOntology defines 'is agent in' given that the definition of the relation makes reference to Agent, defined in that ontology.

**Comment by CarterBeauBenson** [comment]
@alanruttenberg 
I think I understand the pull to this comment, we want to keep the files tidy. That said, the Agent Ontology imports IEO. IEO imports the Geospatial Ontology, and GEO imports ERO. ERO imports BFO. I am probably missing something, but ERO is either directly or indirectly imported into every CCO file and it might be easier to find all of the object properties and annotation properties if they were located in one file. 

I thought the problem with RO was that OBO made changes to it that impacted CCO users so Ron et al. decided to detach CCO from RO.

**Comment by cameronmore** [comment]
Converting to discussion.
